web/src/main/react-dashboard/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "testgrid-dashboard",
   "version": "0.1.0",
   "private": true,
-  "homepage" : "http://ec2-34-238-28-168.compute-1.amazonaws.com:8080/testgrid/v0.9/",
+  "homepage": "http://ec2-34-238-28-168.compute-1.amazonaws.com:8080/testgrid/v0.9/",
```

Comment:
Is it a must to specify the hostname? What is the possibility of stating only the URI?